### PR TITLE
DOCS-3036: Review Get payment method

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on Stoplight, Redocly
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: openapi/components/schemas/PaymentMethod.yaml [hunk 1]
```diff
@@ -1,93 +1,126 @@
...
-          code: '0031'
-          name: ABN Amro Bank
-    type: object
-  icon_urls:
-    properties:
-      large:
-        example: 'https://media.multisafepay.com/img/methods/3x/visa.png'
+      properties:
+        min:
+          description: The minimum amount (in EUR cents) allowed for the payment method.
```
Comment (Kris-MultiSafepay) [inline openapi/components/schemas/PaymentMethod.yaml:11]:
```suggestion
          description: The minimum amount (in eurocents) allowed for the payment method.
```

File: openapi/components/schemas/PaymentMethod.yaml [hunk 2]
```diff
@@ -1,93 +1,126 @@
...
+          type: integer
+          example: 0
+        max:
+          description: The maximum amount (in EUR cents) allowed for the payment method.
```
Comment (Kris-MultiSafepay) [inline openapi/components/schemas/PaymentMethod.yaml:15]:
```suggestion
          description: The maximum amount (in eurocents) allowed for the payment method.
```

File: openapi/paths/payment-methods.yaml [hunk 3]
```diff
@@ -1,170 +1,43 @@
...
-                        type: string
-                      type: array
-                  type: object
-                type: array
               success:
-                type: boolean
-            type: object
-      description: ''
+                $ref: '../components/schemas/Success.yaml'
+              data:
```
Comment (Kris-MultiSafepay) [inline openapi/paths/payment-methods.yaml:39]:
```suggestion
              data:
                 description: Array of payment methods
                 type: array
                 items: 
                   $ref: '../components/schemas/PaymentMethod.yaml'
```
An array of payment methods is returned

File: openapi/paths/payment-methods.yaml [hunk 4]
```diff
@@ -1,170 +1,43 @@
...
+                $ref: '../components/schemas/PaymentMethod.yaml'
```
Comment (Kris-MultiSafepay) [inline openapi/paths/payment-methods.yaml:40]:
```suggestion
```

File: openapi/paths/payment-methods@{payment_method_id}.yaml [hunk 5]
```diff
@@ -1,65 +1,31 @@
 get:
-  description: ''
-  operationId: GetCustomerPaymentMethod
+  description: Get a customer's direct debit details.
```
Comment (Kris-MultiSafepay) [inline openapi/paths/payment-methods@{payment_method_id}.yaml:2]: I believe something might've gone wrong here. The request is to retrieve an individual payment methods by it's identifier (as path parameter).

**Review by Kris-MultiSafepay — COMMENTED** [review]
Couple of tweaks, but this will need a another review based on recent learnings from dev anyway, so it doesn't need to be perfect.

File: openapi/paths/payment-methods@{payment_method_id}.yaml (hunk 5 above)
Comment (FrancescaNCook) [reply openapi/paths/payment-methods@{payment_method_id}.yaml:2]: @Kris-MultiSafepay Yeah, this is the one I was talking about the other day. I think the example was wrong and it made it really difficult to understand the request!

Comment (FrancescaNCook) [reply openapi/paths/payment-methods@{payment_method_id}.yaml:2]: @Kris-MultiSafepay Yeah, this is the one I was talking about the other day. I think the example was wrong and it made it really difficult to understand the request!

Comment (FrancescaNCook) [reply openapi/paths/payment-methods@{payment_method_id}.yaml:2]: @Kris-MultiSafepay Yeah, this is the one I was talking about the other day. I think the example was wrong and it made it really difficult to understand the request!

Comment (FrancescaNCook) [reply openapi/paths/payment-methods@{payment_method_id}.yaml:2]: @Kris-MultiSafepay Yeah, this is the one I was talking about the other day. I think the example was wrong and it made it really difficult to understand the request!

**Pull request merged**